plugin/lspconfig.vim

```diff
@@ -2,6 +2,9 @@ if exists('g:lspconfig')
   finish
 endif
 let g:lspconfig = 1
+if !has("nvim-0.5")
+  finish
```

Comment:
This is not needed. The conventional pattern with Vim plugins, is to `let g:lspconfig = 1` in your configuration, if you have need for this.

Reply:
Didn’t really think of that, seems fine. Feel free to close.